reactive/kubernetes_control_plane.py

```diff
@@ -2463,7 +2463,8 @@ def configure_apiserver():
     elif is_state("endpoint.azure.ready"):
         api_opts["cloud-provider"] = "azure"
         api_opts["cloud-config"] = str(api_cloud_config_path)
-        feature_gates.append("CSIMigrationAzureDisk=false")
+        if get_version("kube-apiserver") < (1, 25, 0):
```

Comment:
You need this check for the other calls as well right? [here](https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/dda9eecdece780956da91bf052371b4ea11f35c6/reactive/kubernetes_control_plane.py#L2647), [here](https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/blob/dda9eecdece780956da91bf052371b4ea11f35c6/reactive/kubernetes_control_plane.py#L2679)